R/rmItem.R

```diff
@@ -66,7 +68,9 @@ rmItem <- function(dgeObj, itemName){
 rmItems <- function(dgeObj, items){
     assertthat::assert_that(!missing(dgeObj),
                             !missing(items),
-                            "DGEobj" %in% class(dgeObj))
+                            msg = "Be sure to specify both a DGEobj and a character vector or list of items to remove.")
```

Comment:
Another be sure instance, I won't point these all out please fix all the same way.